doc/user_guide/samfire.rst

```diff
@@ -0,0 +1,149 @@
+SAMFire
+*******
+
+.. versionadded:: 1.0
+    SAMFire
+
+SAMFire (Smart Adaptive Multi-dimensional Fitting) is an algorithm created to
+avoid the starting value (or local / false minima) problem, which often arises
+when fitting multi-dimensional datasets.
+
+The algorithm will be described in full when accompanying paper is published,
+however we make the implementation available before that.
+
+The idea
+--------
+
+The main idea of SAMFire is to change two things, when compared to the
+traditional way of fitting datasets with many dimensions in the navigation
+space:
+
+ #. Pick a more sensible pixel fitting order.
+ #. Calculate the pixel starting parameters from already fitted parts of the
+     dataset.
+
+Both of these aspects are linked one to another and are represented by two
+different strategy families that SAMFfire uses while operating.
+
+Strategies
+**********
+
+While operating, SAMFire uses a list of strategies to determine how to select
+next pixel and estimate its starting parameters. Only one strategy is used at a
+time. Next strategy is activated when no new pixels are able to fit using the
```

Comment:
How about "The next strategy is chosen from the list when no new pixels can be fitted with the current strategy."